app/views/results/marker/_update_progress_bar.js.erb

```diff
@@ -0,0 +1,13 @@
+<script>
+  function update_bar(score, outof) {
+      var width = (score/outof) * 100;
```

Comment:
I realized that there should probably be a guard for when `outof` is 0 (in this case, maybe there should be a full-width bar).